### PR TITLE
[GLPI 10 Bugfixes] Fix Requester Count Display

### PR DESCRIPTION
**Pull request by tsmr**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #11672

### CODE REVIEW

**Review by orthagh — CHANGES_REQUESTED** [review]
I agree with the miss of the previous feature.
But maybe you can factorize the call of assign/requester information in a single ajax file and just change the title/parameter/etc.

Having a new file for requester doing the same for assign is little too much

File: templates/components/itilobject/actors/field.html.twig [hunk 1]
```diff
@@ -140,27 +148,37 @@
          var option_element = $(`<span class="actor_entry" data-itemtype="${itemtype}" data-items-id="${items_id}" data-actortype="${actorytype}" data-text="${text}">${indent}${icon}${option_text}${actions}</span>`);
 
          // manage ticket information (number of assigned ticket for an actor)
-         if (is_selection && itemtype != "Email" && "{{ actortype }}" == "assign") {
-            var already_assign_element = $(
-               `<span class="assign_infos ms-1" title="{{ __('Number of tickets already assigned') }}"
+         if (is_selection && itemtype != "Email") {
+            if ("{{ actortype }}" == "assign") {
+               var already_element = $(
```
Comment (trasher) [inline templates/components/itilobject/actors/field.html.twig:153]: Variable name is strange.
```suggestion
               var existing_element = $(
```

**Comment by trasher** [comment]
And also, you have CS fails to fix. Appart of that, sounds correct to me that me.

**Review by trasher — CHANGES_REQUESTED** [review]
See previous comments

**Review by trasher — APPROVED** [review]

**Review by orthagh — CHANGES_REQUESTED** [review]
Last thing, rename ticketinformation.php to actorinformation.php

Functional review ok

**Review by orthagh — APPROVED** [review]

**Pull request merged**